README.md

```diff
@@ -83,3 +83,15 @@ $ docker run --rm -e LS_ACCESS_TOKEN -e OTEL_EXPORTER_OTLP_TRACES_ENDPOINT -e TO
 ```
 
 When building with Docker, you need to re-run both steps for any code *or* config changes. If you run into errors while building, please open [an issue](https://github.com/lightstep/telemetry-generator).
+
+## Publishing a Release
+These steps enable a new Docker image to be available with `docker pull ghcr.io/lightstep/telemetry-generator:<tag>`
+
+0. Make whatever changes you're ready to do, add into a release
+1. Update VERSION in the file `VERSION`
+2. Update CHANGELOG.md
+3. Create PR, get approvals, merge changes
+4. Run `make add-tag` 
+    * (This will run `git tag` under the hood using the version number in VERSION)
+5. Run `make push-tag`
+    & (This will push the tags to Github. **THIS** is the operation that will kick off the GHA workflow, build  and push a new image out to GHCR.io)
```

Comment:
```suggestion
    * (This will push the tags to Github. **THIS** is the operation that will kick off the GHA workflow, build  and push a new image out to GHCR.io)
```